docs/security-audit-2026-04-30.md

```
@@ -0,0 +1,189 @@
+| `portal-api-auditor` | `astro-app/src/pages/portal/api/**`, `astro-app/src/pages/api/portal/**` | Clean |
+| `public-api-auditor` | `astro-app/src/pages/api/{subscribe,auth/[...all],events/[slug].ics}.ts` | 2 MEDIUM |
+| `auth-core-auditor` | `auth-config.ts`, `auth-client.ts`, `db.ts`, `drizzle-schema.ts`, `middleware.ts`, `migrations/**` | 2 HIGH, 5 MEDIUM |
+| `render-xss-auditor` | `layouts/**`, `pages/**/*.astro`, `components/{portal,portable-text,blocks}/**` | 2 MEDIUM |
+| `sanity-auditor` | `studio/**`, `astro-app/src/lib/queries/**`, all `defineQuery`/`groq` usage | 1 HIGH (1 dup HIGH consolidated) |
+| `cf-config-worker-auditor` | `wrangler.jsonc`, `astro.config.mjs`, `*-worker/`, `discord-bot/`, `platform-api/`, `scripts/**`, `astro-app/src/actions/**` | Clean |
+
+---
+
+## HIGH-severity findings
+
+### H-1: Sanity Studio admin bearer token shipped in browser bundle
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)